client/components/data/query-post-likes/index.jsx

```diff
@@ -30,13 +41,33 @@ class QueryPostLikes extends Component {
 		}
 	}
 
-	request( { requestPostLikes: rpl, siteId, postId } ) {
-		rpl( siteId, postId );
+	request( {
```

Comment:
we need to call this on some sort of `setInterval` that we setup on mount and remove on dismount